rusttorney-server/src/server.rs

```diff
@@ -114,35 +222,50 @@ impl<'a> AOServer<'a> {
     pub async fn run(&mut self) -> anyhow::Result<()> {
         use futures::StreamExt;
 
+        log::debug!("Beginning migration...");
         self.migrate().await?;
 
         log::info!("Starting up the server...");
-        let addr = format!("127.0.0.1:{}", self.config.general.port);
+        let addr = format!("0.0.0.0:{}", self.config.general.port);
```

Comment:
I still think that the address should be configurable (not only the port but the whole address)